src/DynamicImage/widget/DynamicImage.js

```diff
@@ -20,7 +20,7 @@ define([
 
         postCreate: function () {
             logger.debug(this.id + ".postCreate");
-            this._updateRendering();

```

Comment:
This is useless. ``postCreate`` will call ``_updateRendering()``, but since the ``this._contextObj`` is empty, it will show a warning that the object is ``null``. That's why we only call ``_updateRendering`` on ``update``, so it can tell the ui when it is finished rendering (by calling ``callback``)